server/models/Person.ts

```diff
@@ -0,0 +1,14 @@
+export default interface Person {
```

Comment:
This represents the relationship between a person on probation and the staff member managing them, I think it would be better to name this `PersonManager` instead, what do you think?

Reply:
Yes sure, I'll change 🙌